### PR TITLE
added non-external-id statement count as a signal

### PR DESCRIPTION
**Pull request by guergana**

### CODE REVIEW

**Comment by micgro42** [comment]
Could we adjust its name to include `non-external-id` in some way? Because currently it is named in a way that would imply it duplicates what we do in [wikibase.py#L5](https://github.com/wikimedia/articlequality/blob/581a2aaa14eecb692588956a66bbdde74a278fc1/articlequality/feature_lists/wikibase.py#L5): count all the statements of all the properties.

File: articlequality/feature_lists/wikidatawiki.py [hunk 1]
```diff
@@ -161,6 +172,8 @@ def _process_important_description_translations(item_descriptions):
 unique_references_count = aggregators.len(unique_references)
 "`int` : A count of unique sources in the revision"
 
+non_external_statements_count = aggregators.len(non_external_statements)
+"`int` : A count of all statements in the revision"
```
Comment (mariushoch) [inline articlequality/feature_lists/wikidatawiki.py:176]: Not quite accurate

File: articlequality/feature_lists/wikidatawiki.py [hunk 2]
```diff
@@ -148,6 +148,17 @@ def _process_important_description_translations(item_descriptions):
...
     depends_on=[wikibase_.revision.datasources.descriptions])
 
+def _process_non_external_statements(entity):
+    return [statement
+            for pid, statements in entity.properties.items()
+            if pid in property_datatypes.NONEXTERNAL_IDENTIFIERS
+            for statement in statements]
+
+non_external_statements = Datasource(
+    name + ".revision.statements",
```
Comment (mariushoch) [inline articlequality/feature_lists/wikidatawiki.py:158]: Should be the same as the var name

File: articlequality/feature_lists/wikidatawiki.py [hunk 3]
```diff
@@ -148,6 +148,17 @@ def _process_important_description_translations(item_descriptions):
     _process_important_description_translations,
     depends_on=[wikibase_.revision.datasources.descriptions])
 
+def _process_non_external_statements(entity):
+    return [statement
+            for pid, statements in entity.properties.items()
+            if pid in property_datatypes.NONEXTERNAL_IDENTIFIERS
+            for statement in statements]
+
+non_external_statements = Datasource(
```
Comment (mariushoch) [inline articlequality/feature_lists/wikidatawiki.py:157]: I think `non_external_id_statements` would be even clearer here.

**Review by mariushoch — CHANGES_REQUESTED** [review]

File: articlequality/feature_lists/wikidatawiki.py (hunk 1 above)
Comment (guergana) [reply articlequality/feature_lists/wikidatawiki.py:176]: You are right!

**Review by mariushoch — APPROVED** [review]

**Pull request merged**